exercises/accumulate/.editorconfig

```diff
@@ -0,0 +1,143 @@
+###############################
+# Core EditorConfig Options   #
+###############################
+
+; This file is for unifying the coding style for different editors and IDEs.
+; More information at:
+; https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference?view=vs-2017
+; https://docs.microsoft.com/en-us/visualstudio/ide/create-portable-custom-editor-options?view=vs-2017
+
+root = true
```

Comment:
Should each of the exercises mark themselves as a root config?

Reply:
I think they do, as otherwise the editorconfig format specifies that it will keep on looking for .editorconfig files in the directory's parents, until one is found with `root = true`. In practice, omitting the `root = true` in the exercise `.editorconfig` means that if two/three directories up one would have an `.editorconfig` that adds an unwanted or new formatting feature. My preference would be to include the `root = true`, to prevent situations where the formatting is somehow incorrectly applied due to some parent `.editorconfig` file.

Reply:
Naive question: Does that mean you could put one `.editorconfig` with `root = true` in the `Exercism/csharp` directory, instead of one per `Exercism/csharp/exercise`?

If doing that works without side-effects, it would also solve the maintenance aspect.

Reply:
@Mehni yes that's correct for people that check out the entire solution. The problem is that when students download a specific exercise they're not going to receive the everything in the `Exercism/csharp` directory on the `Exercism/csharp/EXERCISE` directory.

Reply:
Unfortunately, there is no functionality to have track specific, cross-exercise files.